client/extensions/woocommerce/state/data-layer/customers/index.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { customersFailure, customersReceive } from 'woocommerce/state/sites/customers/actions';
+import request from 'woocommerce/state/sites/http-request';
+import { WOOCOMMERCE_CUSTOMERS_REQUEST } from 'woocommerce/state/action-types';
+
+export const fetch = action => {
+	const { siteId, searchTerm } = action;
+	return request( siteId, action ).get( `customers?search=${ searchTerm }&per_page=50` );
```

Comment:
Any use case where we might want to set [other query args](http://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-customers) here down the line - like maybe a paginated list of customers? I guess we can always update if needed later.

Reply:
Do we need to url encode `searchTerm` or is that handled by the data layer?

Reply:
Yeah, I'm leaning to "update later", since I don't think there's anything on the roadmap that'll need customers.